src/components/Engines/Engines.test.tsx

```diff
@@ -1,71 +1,60 @@
 import React from 'react';
 import { mount } from 'enzyme';
```

Comment:
I would suggest to use `@testing-library/react` as some components are already using.  what do you think @juanpicado ?

Reply:
Yes, agree, `enzyme` is not friendly with hooks yet, so testing-librarty is the way, not mandatory, only if the PR owner agree, it can be a new PR as well.

Reply:
When doing a refactor I prefer to keep the tests the same as much as possible, changing the tests at the same time invalidates their purpose (guarding against regressions).

I definitely prefer RTL, will be happy to open a PR for that once this is merged